test-it/karma/javascriptinvoke/ReturnDataTypes.Test.js

```diff
@@ -212,6 +292,14 @@ describe('A JavaScript function invoke', function () {
             expect(viPathParser('returnSingle')).toBe(3.0);
             expect(viPathParser('returnDouble')).toBe(6.0);
             expect(viPathParser('returnString')).toBe('National Instruments');
+            expect(viPathParser('returnInt8Array')).toEqual([1, 2, 3]);
```

Comment:
I am using toEqual here because toBe uses === comparison which is not sufficient for array comparison. Also note that I am comparing the returned JSON to a regular array and not a TypedArray - I think this is because the JSON we get at this point is separate from the actual typed array type. I'm not completely sure on this, so let me know if that is the case or not.